pkg/audit/cursor.go

```diff
@@ -43,17 +43,16 @@ func NewCursor(metainfo *metainfo.Service) *Cursor {
 }
 
 // NextStripe returns a random stripe to be audited
```

Comment:
I would add some doc information regarding when `more` should be considered or not when error is returned and also the case when the stripe can be `nil` and `error` too, but `more` may be true

Reply:
done